Atomic/scan.py

```diff
@@ -96,6 +97,9 @@ def set_scanner():
 
         scan_type = self.get_scan_type()
 
+        if self.args.profile_id is not None and scan_type != "configuration_compliance":
```

Comment:
This check is intended to remind user that he specifies a profile for configuration compliance scan, but he's probably using CVE scan. It is not necessary to do here, so yes, I can leave the decision up to openscap daemon.